### PR TITLE
developing: clarify how to update docker image

### PR DESCRIPTION
**Pull request by NunoAlexandre**
It wasn't clear for me where to get the docker image tag from after setting a non-existent one.

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -43,38 +43,46 @@ replace image tag (last part after `:`) with a nonexistent tag
...
+  **What happens on the build agent:** no docker image
+  can be found for the given tag, therefore the agent will
+  run the full pipeline and save the docker image under a
+  tag associated with the current commit ID.`
+
+3. Copy the current commit ID pushed in step 2. and set it as the new docker image tag.
 
 Example:
 ```
 DOCKER_IMAGE: gcr.io/opensourcecoin/radicle-registry/ci-base:does_not_exist
```
Comment (CodeSandwich) [inline DEVELOPING.md:75]:
```suggestion
DOCKER_IMAGE: gcr.io/opensourcecoin/radicle-registry-eth/ci-base:does_not_exist
```

File: DEVELOPING.md [hunk 2]
```diff
@@ -43,38 +43,46 @@ replace image tag (last part after `:`) with a nonexistent tag
...
 ```
+
 to
+
 ```
 DOCKER_IMAGE: gcr.io/opensourcecoin/radicle-registry/ci-base:e8c699d4827ed893d8dcdab6e72de40732ad5f3c
```
Comment (CodeSandwich) [inline DEVELOPING.md:81]:
```suggestion
DOCKER_IMAGE: gcr.io/opensourcecoin/radicle-registry-eth/ci-base:e8c699d4827ed893d8dcdab6e72de40732ad5f3c
```

**Review by CodeSandwich — APPROVED** [review]
Thank you!

**Comment by NunoAlexandre** [comment]
@CodeSandwich I am afraid I pushed the changes to improve the structure during your review. Sorry, my bad. Pushed your points btw.

**Pull request merged**